apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -211,6 +218,87 @@ export class TeacherFeedback extends Component {
     return feedbackUnchanged;
   };
 
+  getFriendlyDate = feedbackSeen => {
+    const today = new Date();
+    const dateFeedbackSeen = new Date(feedbackSeen);
+
+    //javascript months begin counting at 0
+    const [todayM, todayD, todayY] = [
+      today.getMonth() + 1,
+      today.getDate(),
+      today.getFullYear()
+    ];
+
+    const [m, d, y] = [
+      dateFeedbackSeen.getMonth() + 1,
+      dateFeedbackSeen.getDate(),
+      dateFeedbackSeen.getFullYear()
+    ];
+    if (todayM === m && todayY === y) {
+      if (todayD === d) {
+        return 'today';
+      } else if (todayD === d + 1) {
+        return 'yesterday';
+      }
+    } else {
+      const feedbackYear = (y + '').substr(2, 2);
```

Comment:
You can accomplish this with `y.toString().substr(2, 2);` - this way, the conversion is more explicit